protocol/rpcprovider/rpcprovider.go

```diff
@@ -644,6 +670,9 @@ rpcprovider 127.0.0.1:3333 COS3 tendermintrpc "wss://www.node-path.com:80,https:
 					shardID,
 					rewardsSnapshotThreshold,
 					rewardsSnapshotTimeoutSec,
+					enableRelaysHealth,
+					relaysHealthInterval,
```

Comment:
kinda ugly, could be merged into a struct with all relevant fields to the health monitoring.

Reply:
Fixed in 356d7ddbf4f895774d6f0bc949eb768ff16754ee